app/locations/views/locations.njk

```diff
@@ -34,6 +34,14 @@
               </a>
             </li>
           {% endfor %}
+          {% for region in regions %}
```

Comment:
If we're combing both lists it might be better to list the regions first as there are likely to be less of them.

Some users who may have access to regions could have access to a lot of locations via NOMIS "caseloads" so the regions will appear a long way down the page.

I wonder if it might be nicer to use separate `ul`s too as the data within the list is technically different?